app/jobs/set_participant_start_date_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# This is a one off job to update all of the induction start dates from DQT for
+# all of the 2021/2022 ECTs that do not currently have them.
+# It runs in batches of 200 as we have a 300 lookup per minute limit on the API
+# There are approx 57600 participants in this state currently
+class SetParticipantStartDateJob < ApplicationJob
+  def perform
+    ParticipantProfile::ECT
+      .joins(schedule: :cohort)
+      .eligible_status
+      .where(cohort: { start_year: [2021, 2022] })
```

Comment:
I don't think we should trust the cohort pointed by the ParticipantProfile but their latest induction record instead?

Reply:
Thought the `schedule.cohort` was about as authoritative as it got (shouldn't those match?)

Reply:
`Participants::SyncDqtInductionStartDate` probably fits better for this purpose. 
It actually takes that into account and only swap cohorts for the agreed participants.
That means it will actually only update the induction start date to pre_2023 participants (exactly what you need).

Reply:
@ltello yes you might be right, I thought that service did more than just update the induction start date and I definitely don't want to update anything other than that as this is supposed to be a non-intrusive back-fill task. In addition because the query is only including `eligible` ECTs they should all have induction start dates that we've checked so no shifting should be necessary.

Reply:
@tonyheadford what I am not getting is why we need a new service to sync the induction start dates of some participants from DQT when we carefully analised and made decisions on how to do it a few weeks ago and the result was the existing service `Participants::SyncDqtInductionStartDate`. 
Why not use it?
The participants you are trying to update now were already considered in that analysis and the existing service is doing exactly what your new code will do.

Reply:
@ltello as mentioned above I (wrongly) thought the `SyncDQTInductionStartDate` contained more logic than it does and would not work for this but I am planning to switch to it instead when I update this PR.

Reply:
@tonyheadford sorry, I misread your message. I (mis)understood you didn't want to use the existing service because it does more than you want to do (which is true but not for the participants you are dealing with in this PR).